packages/contracts-bedrock/deploy-config/espresso-demo.json

```diff
@@ -0,0 +1,54 @@
+{
```

Comment:
For the other devnets we don't commit this generate file, only the template. I think we should `git rm` this and add it to `packages/contracts-bedrock/.gitignore`. Unless there's a specific reason to version control it